framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -515,6 +515,11 @@ protected void setParentChecker(SourceChecker parentChecker) {
         this.parentChecker = parentChecker;
     }
 
+    /** @return the parent checker of the current checker */
+    public SourceChecker getParentChecker() {
+        return this.parentChecker;
+    }
+
     /** Invoked when the current compilation unit root changes. */
```

Comment:
@xingweitian Can you add the missing `@param` to make the tests pass. You didn't touch that method, but it is "close" to what you added.